src/RequestForm.js

```diff
@@ -208,6 +208,7 @@ class RequestForm extends React.Component {
       shouldValidateInstanceId: false,
       isInstanceIdBlur: false,
       validatedInstanceId: null,
+      isItemIdRequest: Boolean(this.props.query.itemId),
```

Comment:
@artem-blazhko is `query` prop always present on `RequestForm` or would it be safer to do `this.props.query?.itemId` here?

Reply:
Thank you @mkuklis.
I am not definitely sure that `query` is always present. But anyway I added conditional property.